lib/src/locale_gen_sb_writer.dart

```diff
@@ -160,8 +160,11 @@ class LocaleGenSbWriter {
           LocaleGenParser.parseDefaultLanguageLocale(params.defaultLanguage))
       ..writeln()
       ..writeln('  static const _supportedLocales = [');
-    params.languages.forEach((language) =>
-        sb.writeln(LocaleGenParser.parseSupportedLocale(language)));
+    params.languages
+      ..sort((a1, a2) =>
+          a1 == params.defaultLanguage || a2 != params.defaultLanguage ? 0 : 1)
```

Comment:
This does not sort the remaining languages, it just moves the default to the front. There are better ways to code this behaviour

Reply:
but that is what we want. we only want to have de default language at position 0. the position of the other languages is not important

Reply:
Then why not create a version that does this instead of abusing sort